Gems/Atom/Tools/MaterialCanvas/Code/Source/Window/MaterialCanvasGraphView.cpp

```diff
@@ -11,14 +11,51 @@
 
 namespace MaterialCanvas
 {
+    void MaterialCanvasGraphConstructPresets::InitializeConstructType(GraphCanvas::ConstructType constructType)
+    {
+        if (constructType == GraphCanvas::ConstructType::NodeGroup)
+        {
+            auto presetBucket = static_cast<GraphCanvas::NodeGroupPresetBucket*>(ModPresetBucket(GraphCanvas::ConstructType::NodeGroup));
+            if (presetBucket)
+            {
+                presetBucket->ClearPresets();
+
+                const AZStd::map<AZStd::string, AZ::Color> defaultPresetGroups = {
```

Comment:
We're these colors derived from something somewhere, or is there a notion of "theming" that our UI supports at all? I'm wondering if we should considering setting things up so that UI folks can make changes to things like this in data without needing a code check-in in the future. (doesn't have to be this PR, but a general "plan" for how to extract form from function to that end might be useful)

Reply:
These colors were copied from script canvas. I'm preparing a follow up PR that passes in this list of names and colors and allows the user to configure them from the settings dialog. Either way, these are just the defaults and the user can create new presets based on other groups, comments, nodes in the scene.